syntax = "proto3";
import "google/protobuf/any.proto";

option go_package = "../common";

message ImageData {
    double identifier = 1;
    string data = 2;
}